{"uid":"d077d78c1257a94b","name":"test_update_team_with_empty_name_returns_400","fullName":"test.teams.test_put_team#test_update_team_with_empty_name_returns_400","historyId":"6ae4b307c4743ab6f66268eae49e8c27","time":{"start":1726090356027,"stop":1726090356205,"duration":178},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"66e20c74000c495a6\",\"name\":null,\"deleted\":false,\"positionList\":[\"Designer\",\"Product Owner\",\"Dev\"],\"createdAt\":\"2024-09-11 21:32:36\",\"modifiedAt\":\"2024-09-11 21:32:36\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}","statusTrace":"setup_teardown_team = {'createdAt': '2024-09-11 21:32:36', 'deleted': False, 'id': '66e20c74000c495a6', 'layoutSetId': '66d8f8ff4f1c23bca', ...}\n\n    @pytest.mark.updateteam\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_update_team_with_empty_name_returns_400(setup_teardown_team):\n        team1 = setup_teardown_team\n        url = EndpointTeams.get_team_id(team1['id'])\n        data = create_team_data(\n            name=\"\")\n        response = TeamService.update_team(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/teams/test_put_team.py:46: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"66e20c74000c495a6\",\"name\":null,\"deleted\":false,\"positionList\":[\"Designer\",\"Product Owner\",\"Dev\"],\"createdAt\":\"2024-09-11 21:32:36\",\"modifiedAt\":\"2024-09-11 21:32:36\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}\n\ncore/assertions/status_code_assertion.py:37: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1726090329569,"stop":1726090329586,"duration":17},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_teardown_team","time":{"start":1726090355842,"stop":1726090356026,"duration":184},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"66e20c74000c495a6\",\"name\":null,\"deleted\":false,\"positionList\":[\"Designer\",\"Product Owner\",\"Dev\"],\"createdAt\":\"2024-09-11 21:32:36\",\"modifiedAt\":\"2024-09-11 21:32:36\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}","statusTrace":"setup_teardown_team = {'createdAt': '2024-09-11 21:32:36', 'deleted': False, 'id': '66e20c74000c495a6', 'layoutSetId': '66d8f8ff4f1c23bca', ...}\n\n    @pytest.mark.updateteam\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_update_team_with_empty_name_returns_400(setup_teardown_team):\n        team1 = setup_teardown_team\n        url = EndpointTeams.get_team_id(team1['id'])\n        data = create_team_data(\n            name=\"\")\n        response = TeamService.update_team(url, data)\n>       AssertionManager.assert_status_code_400(response)\n\ntest/teams/test_put_team.py:46: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n        logger.info(f\"Validating status code 400 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 400, (\n                f\"Expected status code 400, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 400, but got 200. Response text: {\"id\":\"66e20c74000c495a6\",\"name\":null,\"deleted\":false,\"positionList\":[\"Designer\",\"Product Owner\",\"Dev\"],\"createdAt\":\"2024-09-11 21:32:36\",\"modifiedAt\":\"2024-09-11 21:32:36\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}\n\ncore/assertions/status_code_assertion.py:37: AssertionError","steps":[],"attachments":[{"uid":"ae11552507e25dbc","name":"log","source":"ae11552507e25dbc.txt","type":"text/plain","size":4627},{"uid":"59b99e1bbef8a695","name":"stderr","source":"59b99e1bbef8a695.txt","type":"text/plain","size":4846}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[{"name":"setup_teardown_team::0","time":{"start":1726090356211,"stop":1726090356376,"duration":165},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"updateteam"},{"name":"parentSuite","value":"test.teams"},{"name":"suite","value":"test_put_team"},{"name":"host","value":"fv-az1205-438"},{"name":"thread","value":"2065-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.teams.test_put_team"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["updateteam","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"d077d78c1257a94b.json","parameterValues":[]}